powershell/llcsharp/operation/method.ts

```diff
@@ -482,12 +478,6 @@ export class CallMethod extends Method {
             const content = Local('content', new LiteralExpression(`await ${response.value}.Content.ReadAsStringAsync()`));
             yield content;
 
-            yield 'await waiting;';
-
-            yield EOL;
-            yield '// check for cancellation';
-            yield `if( ${$this.opMethod.contextParameter}.Token.IsCancellationRequested ) { return; }`;
```

Comment:
Need to double check if this can be removed

Reply:
Is it better we start line#456 without await and then await in line#480?

Reply:
I just did some test. The cost of await earlier is 0~2 ms, so I think it's fine.